app/generate-badges.py

```diff
@@ -10,18 +10,24 @@
 from lxml import etree
 
 NUMBER_OF_BADGES_PER_PAGE = 8
+# Initializing and setting global variable `NUMBER_OF_BADGES_PER_PAGE`
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
+# Get root folder
 UPLOAD_FOLDER = os.path.join(APP_ROOT, 'static/uploads')
+# Get Upload Folder
```

Comment:
It is self explanatory, seeing it in constant notation. Try to not use words like `Get`.